src/main/java/racingcar/InputView.java

```diff
@@ -0,0 +1,24 @@
+package racingcar;
+
+import java.util.Scanner;
+
+public class InputView {
+    private static final String ankCarCount = "자동차 대수는 몇 대 인가요?";
+    private static final String askTryCount = "시도할 회수는 몇 회 인가요?";
+
+    private final Scanner scanner;
```

Comment:
생성과 동시에 정적 상수로 초기화 해도 괜찮을 것 같습니다 😄 
그럼 기본 생성자는 없어도 되니까요 😄 
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```

Reply:
괜한짓을 했군요ㅜㅜ 수정했습니다!